pages/_app.tsx

```diff
@@ -3,6 +3,14 @@ import { siteMetaConfig } from '@/config/site.config';
 import '@/styles/globals.css';
 import type { AppProps } from 'next/app';
 import Head from 'next/head';
+import PostHog, { PostHogConfig } from 'posthog-js';
```

Comment:
Please check this section. It's failing deployment

<img width="1069" alt="Screenshot 2024-02-27 at 23 27 32" src="https://github.com/hackmamba-io/docs/assets/35299371/043313cd-3821-4bb7-b84a-e761c83f21d8">

Reply:
Alright